### DISCUSSION TITLE
Error with php 7.3

### DISCUSSION THREAD

**Discussion by a6e6s** [livewire/livewire#5467] [1. Help] [answered]
livewire isn't work on php 7.3
I was forced to downgrade my laravel8 app to php 7.3 from 7.4 and after that livewire stop working I did check every thing
even with clean Laravel 8/livewire App installation from scratch, with the simplest component,  its return popup with [404 NOT FOUND]
asset path is fine the only thing is different is the php version

**Comment by a6e6s** [comment]
On the browser console it shows error in this function 

// Forward the query string for the ajax requests.

        fetch(
            `${window.livewire_app_url}/livewire/message/${payload.fingerprint.name}`,
            {
                method: 'POST',
                body: JSON.stringify(payload),
                // This enables "cookies".
                credentials: 'same-origin',
                headers: {
                    'Content-Type': 'application/json',
                    'Accept': 'text/html, application/xhtml+xml',
                    'X-Livewire': true,

                    // set Custom Headers
                    ...(this.headers),

                    // We'll set this explicitly to mitigate potential interference from ad-blockers/etc.
                    'Referer': window.location.href,
                    ...(csrfToken && { 'X-CSRF-TOKEN': csrfToken }),
                    ...(socketId && { 'X-Socket-ID': socketId })
                },
            }
        )

**Comment by a6e6s** [comment, marked as answer]
I did found the solution i did copy my vendor/livewire folder from the app on php7.4
and replace the folder on the php7.3 app and now its working fine

**Comment by joshhanley** [comment]
@a6e6s there was a bug released in v2.10.8. I have submitted a PR #5490 with a fix. Hope this helps!
